### PR TITLE
Feature/ufs 234 funders opp setup

### PR DESCRIPTION
**Pull request by torchiad**

### CODE REVIEW

File: src/components/SetupFunders/SetupFunders.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+    return (
+        <div>
+            <h1>Funders</h1>
+            {funders.map(funder => (
+                <Checkbox
+                    key={funder.name}
+                    checked={funderList[funder.name]}
+                    onChange={(event: SyntheticEvent<HTMLInputElement>) => {
+                        console.log(event.currentTarget);
```
Comment (ascorbic) [inline src/components/SetupFunders/SetupFunders.tsx]: You can remove this log

File: src/components/SetupFundersPage/SetupFundersPage.story.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { SetupFundersPage } from "./index";
+
+storiesOf("Components|SetupFundersPage", module).add("Default", () => (
```
Comment (ascorbic) [inline src/components/SetupFundersPage/SetupFundersPage.story.tsx]: This story doesn't work – I recommend removing stories on the graphql pages until we work out a way of mocking it.

File: src/components/SetupFundersPage/SetupFundersPage.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+    const opportunity = data;
+
+    const addFunderMutation = useMutation<
+        {},
+        UpdateOpportunityMutationVariables
+    >(UPDATE_OPPORTUNITY);
+
+    const addFunder = useCallback(
+        async (name: Array<string>) => {
```
Comment (ascorbic) [inline src/components/SetupFundersPage/SetupFundersPage.tsx]: Change type to `string[]`

File: src/components/SetupFunders/SetupFundersTest.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import React from "react";
```
Comment (ascorbic) [inline src/components/SetupFunders/SetupFundersTest.tsx]: This file needs to be renamed to SetupFunders.test.tsx, as it's not being picked up by Jest. This had been updated in the latest hygen templates.

File: src/components/SetupFundersPage/SetupFundersPageTest.tsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
```
Comment (ascorbic) [inline src/components/SetupFundersPage/SetupFundersPageTest.tsx]: This file needs to be renamed to SetupFundersPage.test.tsx, as it's not being picked up by Jest. However I'm not convinced it'll work.

**Review by ascorbic — CHANGES_REQUESTED** [review]

**Review by ascorbic — CHANGES_REQUESTED** [review]
The tests are still failing on this. Run `yarn test` and fix the issues. You should remove the test on graphql pages for now.

**Comment by torchiad** [comment]
> The tests are still failing on this. Run `yarn test` and fix the issues. You should remove the test on graphql pages for now.

@ascorbic I've removed those tests and added updated snapshots. 3 passes and no failures.

We do get a warning: "A "describe" callback must not return a value.
          Returning a value from "describe" will fail the test in a future version of Jest."

**Review by ascorbic — APPROVED** [review]

**Pull request merged**